guides/pipx.md

```diff
@@ -0,0 +1,85 @@
+# Installing and using pipx for command-line Python tools
+
+[pipx](https://github.com/pipxproject/pipx) is a utility for installing and running Python applications in isolated environments. This means it allows you to [install command-line tools like `tox`](#installing-command-line-tools-with-pipx) in their own virtualenv that pipx manages for you using simple, `pip`-like commands. 
```

Comment:
The second link here (to `#installing-command-line-tools-with-pipx`) seems excessive, since the same link is provided in the list just below.

Reply:
I think I'm going to leave it for now, since it's a difference between being the context of the sentence verses the overall page TOC.